mep3_navigation/params/nav2_params_big.yaml

```diff
@@ -274,6 +274,12 @@ planner_server:
     use_sim_time: True
     planner_plugins: ["GridBased"]
     GridBased:
+      plugin: 'nav2_navfn_planner/NavfnPlanner'
+      use_astar: True
+      allow_unknown: True
+      use_final_approach_orientation: False
+      tolerance: 0.001
```

Comment:
Nema mi smisla da je tolerance u globalnom planeru 0.001 ako je rezolucija costmape 0.008. Sta je problem sa Smac?

Reply:
To nisam spomenuo u opisu PR-a :/ Putanja je nekako cudno isprekidana i cini mi se da je ovaj malo konzistentniji. Ali prepusticu to tebi da uporedis. Glavni cilj PR-a je bio da dodam custom BT.

Reply:
Ok, ja cu u novom PR te sitnice resavati.

Reply:
Za preciznost, s obzirom da nije pucalo smanjio sam na 1mm. Ali nisam siguran da je rezolucija costmape ovdje bitna, treba provjeriti, posto ja msm globalni planer salje `double` tacke i onda posljednja tacka ne mora biti u centru piksela.